sdk/nodejs/types/input.ts

```diff
@@ -29183,7 +29183,7 @@ export namespace iot {
         roleArn: pulumi.Input<string>;
     }
 
-    export interface TopicRuleSns {
+    export interface TopicRuleSn {
```

Comment:
Another issue.

Reply:
Fixed (pinned to `sns`) in https://github.com/pulumi/pulumi-aws/pull/2615/commits/28e1921c4b75c6e88db8281312d4b817ef3e7825.